cdc/owner/feed_state_manager.go

```diff
@@ -111,7 +111,6 @@ func (m *feedStateManager) handleAdminJob() (jobsPending bool) {
 		}
 		m.shouldBeRunning = false
 		jobsPending = true
-		m.patchState(model.StateRemoved)
```

Comment:
Why is this line removed

Reply:
I confirmed with @leoppro , this line is redundant.
```golang
		// remove changefeed info and state
		m.state.PatchInfo(func(info *model.ChangeFeedInfo) (*model.ChangeFeedInfo, bool, error) {
			return nil, true, nil
		})
		m.state.PatchStatus(func(status *model.ChangeFeedStatus) (*model.ChangeFeedStatus, bool, error) {
			return nil, true, nil
		})
```
The above two lines of code will directly delete changefeed.